### ISSUE TITLE
Fading is no more working [duplicate/watched/rated]

### ISSUE THREAD

**Issue by vishal8492** [jaredsohn/flix_plus#21]
Fading is no more working for any of duplicate/watched/rated titles.

**Comment by vishal8492** [comment]
This looks pretty bad , netflix seemed to have changed react_id to not include actual movie/tv_show ids. I'm currently using ids in href which seems to work but I don't like this solution at all. I haven't checked what else is breaking.

**Comment by jaredsohn** [comment]
Thanks for bringing this to my attention.

I agree re: your assessment; I might have to remove some features but I probably won't get to this for another week or so.

**Comment by vishal8492** [comment]
Yeah cool , take your time no rush.

**Comment by EO2** [comment]
Could you grab the movie ID from the title URL and use that instead?
https://www.netflix.com/watch/<0-9>

Or is that a different ID than what is returned from watched list?

It would be great to have the option to totally hide watched titles to free up space.
(hide the div class 'slider-item')
+
Link to display/hide watched shows. (hide watched)
Link to display only watched shows. (hide everything else)

**Comment by jaredsohn** [comment]
@EO2 Yeah, I think I'll probably end up updating this to use the id in the url; I will have to verify that it matches the rated/watched list, though.

The extension used to allow hiding rated/watched/duplicate titles before Netflix has a big UI revamp; the reason we don't do that now is that the page only loads a portion of the images at once so you might see, for example, only three posters before you scroll again. You should be able to see this for yourself if you hide things via the console.

Also, FYI, there is a UI (under configure) for choosing what to do for rated/watched/duplicates - faded, tinted, and normal. In an earlier version of the extension there was a 'hidden' option as well.

**Comment by Zarion** [comment]
Is there a chance that this feature will be fixed? 

Or maybe work on this extension has been abandoned?

**Comment by jaredsohn** [comment]
Sorry for the lack of follow-up; I'll see if I can look at this again this week.

**Comment by vishal8492** [comment]
@jaredsohn I agree, although partial fetching of shows do not happen quite often. It's usually on first load, worst case you initiate page refresh or extension does this. There is no need to abandon this feature, this is  quite important feature.
